core-processor/src/configs.rs

```diff
@@ -82,6 +82,8 @@ pub struct ExecutionSettings {
     pub waitlist_cost: u64,
     /// Reserve for parameter of scheduling.
     pub reserve_for: u32,
+    /// Most recently determined random seed, along with the time in the past since when it was determinable by chain observers.
+    pub random_data: (Vec<u8>, u32),
```

Comment:
And here probably slice.
Seems that this should be in block config

Reply:
> Seems that this should be in block config

The first random data is generated with the message id as subject. Putting this in the block config would not be correct.

Reply:
> > Seems that this should be in block config
> 
> The first random data is generated with the message id as subject. Putting this in the block config would not be correct.

How do you feel about such scheme:
1. Get the seed for a block
2. Throw it on a backend side
3. Append given by contract subject with message_id currently processed
4. Hash it and return

This will decrease amount of hashes, but will keep the same uniqueness for result

Reply:
discussed that we can play around with it in the future.